### ISSUE TITLE
Run Project

### ISSUE THREAD

**Issue by friouimeher1** [bonnetn/dare#2]
...............................

**Comment by fronojono** [comment]
@bonnetn

**Comment by bonnetn** [comment]
Hi fronojono,

did you run the API server Do you have any logs on the API?

**Comment by fronojono** [comment]
Yes Sure ,I follow all the steps ,everything work ,but when Test it in the browser i get this error as screenshot show

**Comment by fronojono** [comment]
![image](https://user-images.githubusercontent.com/27731753/68927370-ea3bfa00-0787-11ea-854e-008c3dc69f4f.png)
![image](https://user-images.githubusercontent.com/27731753/68927510-3850fd80-0788-11ea-9693-2a4b77f19f42.png)
![image](https://user-images.githubusercontent.com/27731753/68927542-4c94fa80-0788-11ea-8e4e-e1936abd36b2.png)
- Run Envoy and mongo Frist 
- Run Backend Side Second 
- Run Frontend Third
